### ISSUE TITLE
Silent failure of `compile-typescript-files.js` utility used by `@nrwl/node:package` builder.

### ISSUE THREAD

**Issue by shirakaba** [nrwl/nx#4658]
## Current Behavior

When running the builder `@nrwl/node:package`, if TypeScript compilation fails, no error is reported.

## Expected Behavior

A stack trace giving the reason for compile failure should be printed out.

A log message should also be printed out, at least informing that there was a failure.

## Steps to Reproduce

Create any standard Node package and induce something that will cause a TypeScript compile failure, for example, a syntax error.

Run the `@nrwl/node:package` builder.

### Failure Logs

The logs contain nothing that indicates failure or completion – just this:

```
Compiling TypeScript files for library yoga...
```

The error "Could not compile Typescript files" (which is passed into the subscriber) did not follow. No stack trace followed, either.

## Suggested fix

After this line, print the stack trace for the user:

https://github.com/nrwl/nx/blob/a97c2426492f641dd93794203cc60796640dc69f/packages/node/src/builders/package/utils/compile-typescript-files.ts#L49

### Environment

```
  nx : Not Found
  @nrwl/angular : Not Found
  @nrwl/cli : 10.3.3
  @nrwl/cypress : Not Found
  @nrwl/eslint-plugin-nx : 10.3.3
  @nrwl/express : Not Found
  @nrwl/jest : 10.3.3
  @nrwl/linter : 10.3.3
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : 10.3.3
  @nrwl/react : Not Found
  @nrwl/schematics : Not Found
  @nrwl/tao : 10.3.3
  @nrwl/web : Not Found
  @nrwl/workspace : 10.3.3
  typescript : 4.0.5
```

**Comment by bekos** [comment]
@shirakaba Seems like this issue is already resolved via 87ded432edf65c3020b69a38a91a5e869243aa4f. Can you please try to upgrade and try again or create a repro repo where I can debug this?

**Comment by bekos** [comment]
Closing as it seems to be fixed, but feel free to reopen in case you are still experiencing the problem and have more info for us 😃
